cmake/DD4hepConfig.cmake.in

```diff
@@ -13,8 +13,16 @@
 # @author Jan Engels, Desy
 ##############################################################################
 
-set ( DD4hep_DIR         "$ENV{DD4hep_DIR}" )
-set ( DD4hep_ROOT        "$ENV{DD4hep_ROOT}" )
+if( DEFINED ENV{DD4hep_DIR} )
+  set ( DD4hep_DIR         "$ENV{DD4hep_DIR}" )
+else()
+  set ( DD4hep_DIR         "@CMAKE_INSTALL_PREFIX@" )
+endif()
+if( DEFINED ENV{DD4hep_ROOT} )
+  set ( DD4hep_ROOT        "$ENV{DD4hep_ROOT}" )
+else()
+  set ( DD4hep_ROOT         ${DD4hep_DIR} )
```

Comment:
I think you want `@CMAKE_INSTALL_PREFIX@` not `${DD4hep_DIR}`

Reply:
You are right, otherwise it is not robust if the user overrides DD4hep_DIR but not DD4hep_ROOT. Fixed.